x/group/spec/messages.md

```diff
@@ -0,0 +1,46 @@
+```go
+// Groups get their own GroupID
+type MsgCreateGroup struct {
+	Signer sdk.AccAddress `json:"signer"`
+	// The Owner of the group is allowed to change the group structure. A group account
+	// can own a group in order for the group to be able to manage its own members
+	Owner  sdk.AccAddress `json:"owner"`
```

Comment:
Chicken and egg issue...
I need to create a group (alex: read "electorate") in order to create a group account (alex: read "election rule").
So, how can I set it as owner on creation of the group.

I would propose a 3 step setup:
* X creates a group and is automatically the owner (owner == signer)
* X creates a group account with a decision policy (eg. 2 of 3) on this group
* X abdicates the thrown and makes the given group account the owner of the group

Those can be 3 messages in one tx and we can do that behind the scenes in the user interface, but in order to avoid some weird causality tangle, I suggest to make them explicitly separate

Reply:
This brings up the point that we need a `MsgUpdateGroup` as well